### PR TITLE
Filling out TBD sections in chapter 4.

### PR DESCRIPTION
**Pull request by jlk**

### CODE REVIEW

**Review by OmerKahani — APPROVED** [review]

File: operator-wg/whitepaper-community-review/README.md [hunk 1]
```diff
@@ -283,8 +282,21 @@ documentation, operator scope, and vulnerability analysis.
 
 #### Transparency and Documentation
 During development of an operator you should have a clear understanding
```
Comment (thschue) [inline operator-wg/whitepaper-community-review/README.md:284]:
```suggestion
During development of an operator a developer should have a clear understanding
```

File: operator-wg/whitepaper-community-review/README.md [hunk 2]
```diff
@@ -283,8 +282,21 @@ documentation, operator scope, and vulnerability analysis.
...
-of how it will work and interface within Kubernetes. You should
-provide your users with some or all of this list:
+of how it will work and interface within Kubernetes. As you shift
+from development to publishing your work, your users should also
+have a clear understanding of what your operator does, and how.
```
Comment (thschue) [inline operator-wg/whitepaper-community-review/README.md:287]:
```suggestion
have a clear understanding of what the operator does, and how.
```

File: operator-wg/whitepaper-community-review/README.md [hunk 3]
```diff
@@ -283,8 +282,21 @@ documentation, operator scope, and vulnerability analysis.
 
 #### Transparency and Documentation
 During development of an operator you should have a clear understanding
-of how it will work and interface within Kubernetes. You should
-provide your users with some or all of this list:
+of how it will work and interface within Kubernetes. As you shift
+from development to publishing your work, your users should also
```
Comment (thschue) [inline operator-wg/whitepaper-community-review/README.md:286]:
```suggestion
from development to publishing the work (operator), users should also
```

File: operator-wg/whitepaper-community-review/README.md [hunk 4]
```diff
@@ -283,8 +282,21 @@ documentation, operator scope, and vulnerability analysis.
...
+have a clear understanding of what your operator does, and how.
+
+You've written something you're proud of, but think of this from
+the end user's point of view: Why should they trust you and your
+operator to run with administrative access on their cluster which
+may be large and costly, or may be handling sensitive information.
+Anything you can do to help this (probably busy) person come up to
```
Comment (thschue) [inline operator-wg/whitepaper-community-review/README.md:293]: Change to third person voice ...

File: operator-wg/whitepaper-community-review/README.md [hunk 5]
```diff
@@ -294,14 +306,33 @@ stay in scope of compliance or you risk vulnerability outside that
 scope.
 * Documented RBAC scopes, threat model, communication ports, API
 calls available
+* Security reporting, disclosure, and incident response processes:
+If someone finds a potential security issue, who should they contact
+and what type of response should they expect?
+* If your project has had security disclosures in the past, listing
```
Comment (thschue) [inline operator-wg/whitepaper-community-review/README.md:312]:
```suggestion
* If the project has had security disclosures in the past, listing
```

File: operator-wg/whitepaper-community-review/README.md [hunk 6]
```diff
@@ -294,14 +306,33 @@ stay in scope of compliance or you risk vulnerability outside that
...
+these disclosures (and their CVE IDs) on a web page is a strong step
+in building trust with your users. Everyone will have security
```
Comment (thschue) [inline operator-wg/whitepaper-community-review/README.md:314]:
```suggestion
in building trust with users. Everyone will have security
```

File: operator-wg/whitepaper-community-review/README.md [hunk 7]
```diff
@@ -294,14 +306,33 @@ stay in scope of compliance or you risk vulnerability outside that
...
+issues at some point - how you handle them displays the maturity
```
Comment (thschue) [inline operator-wg/whitepaper-community-review/README.md:315]:
```suggestion
issues at some point - how they are handled displays the maturity
```

File: operator-wg/whitepaper-community-review/README.md [hunk 8]
```diff
@@ -294,14 +306,33 @@ stay in scope of compliance or you risk vulnerability outside that
...
-### Application Developer ("users")
+A core concept for a developer to consider is if an operator should
+be scoped to run within a sole namespace, or if it should run with
+cluster-wide privileges. While this paper also talks about scoping
+from a user point-of-view, how an operator is designed will weigh
+heavily on the type of security controls which can be applied against
+it in production. It is common to start with lax permissions, and
+intentions to apply security concepts before release; Spending some
+time thinking about the security design of the operator as you begin
+work on it will make this process much easier for you and your
```
Comment (thschue) [inline operator-wg/whitepaper-community-review/README.md:332]:
```suggestion
work on it will make this process much easier for developers and their
```

**Review by thschue — CHANGES_REQUESTED** [review]
Please change to third person voice as far as possible ...

File: operator-wg/whitepaper-community-review/README.md (hunk 4 above)
Comment (jlk) [reply operator-wg/whitepaper-community-review/README.md]: done

**Pull request merged**